docs/manual/manual_ifci_control.rst

```diff
@@ -0,0 +1,61 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _controlling_ifci:
+
+**********************************
+IQUART Flight Controller Interface
+**********************************
+
+Vertiq 2306 modules can be controlled with the IQUART Flight Controller Interface (IFCI) when flashed with the pulsing firmware. A specifc IQUART packet has been created to allow for highspeed control of Vertiq modules. A single packet can include all of the control signals for multiple motors connected to the same UART port on a flight controller. Additionally a telemetry packet can be requested from a specific module every time the packet is sent. A description of the packet is shown in the diagram below.
+
+.. figure:: ../_static/manual_images/pulsing_propeller/Long_Packed_Control_Message.png
+    :align: center
+    :alt: Packed Control Message Description
+
+    Packed Control Message Description
+
+A general description of IQUART packets can be found in :ref:`manual_iquart`.
+
+Each control value is two bytes and can encode either a throttle value, an X value, or a Y value. These values are considered 'Raw' and represent a range of 0.0 to 1.0. The modules can be set to respond to any of the control values by setting the Control Value Index (CVI) in the :ref:`IQUART Flight Controller Interface Client<ifci_message_table>`. Setting 'throttle_cvi' to a number will make it respond to the corresponding control value. For example if throttle_cvi is set to 2 and a packed control message has a control value in index 2 then the module will set its throttle based on the value at that control index. The same is true for x_cvi and y_cvi. The way that the module will respond to the control value for throttle and x/y controls are defined by settings in the :ref:`ESC Propeller Input Parser<epip_message_table>` and :ref:`Pulsing Rectangular Input Parser<prip_message_table>` respectively.
```

Comment:
The modules can read any individual control value in the Packed Control Message though a corresponding Control Value Index (CVI) values (link to thing). 
Kill the "setting 'throttle_cvi' sentence, and go right to "For example, ..."